protocol/avdl/keybase1/common.avdl

```diff
@@ -292,23 +292,24 @@ protocol Common {
 
   /**
     SocialAssertion contains a service and username for that service, that
-    together form an assertion about a user. Resolving an assertion requires
+    together form an assertion about a user. It can either be a social
+    assertion (like "facebook" or "twitter") or a server trust assertion (like
+    "phone" or "email").
+
+    If the assertion is for social network, resolving an assertion requires
     that the user posts a Keybase proof on the asserted service as the asserted
     user.
+
+    For server trust assertion, we have to trust the server.
+
+    TODO: SocialAssertion is historical name for this type, but it's not
+    correct since we do phone/email assertions as well.
```

Comment:
is there a plan to get this todo todone?

Reply:
ok seems like we need to decide one way or the other whether phone numbers can live here and not have this as a todo. might want to ask kbfs

Reply:
@maxtaco can you take a look?

The problem is that we had to shoehorn servertrust phone type invites into `SocialAssertion` record type, to avoid having to change any of the implicit team related types. Refactoring this would require bunch of changes in both client and KBFS.

Reply:
It shouldn't be too bad to change the name of the type since it doesn't affect the wire format at all. And then KBFS can change their types independently.

Reply:
FYI CircleCI will likely fail if you change this without using type aliases, so it'll require a double-repo merge.  But it's fine with me!

Reply:
should we do that as part of this PR or make a new one?

Reply:
@mlsteele OK to merge and then work on a refactor?

Is there anything else I should fix in this PR? Thanks!